sqldelight-core/src/test/table-interface-fixtures/requires-adapter/Person.kt

```diff
@@ -0,0 +1,53 @@
+import com.squareup.Redacted
+import com.squareup.sqldelight.ColumnAdapter
+import java.util.List
+import kotlin.Boolean
+import kotlin.ByteArray
+import kotlin.Int
+import kotlin.String
+
+interface Person {
+  fun _id(): Int
+
+  fun name(): String
+
+  fun last_name(): String?
+
+  fun is_cool(): Boolean
+
+  fun friends(): List<com.sample.Person>?
+
+  fun shhh_its_secret(): @Redacted String
+
+  data class Adapter(internal val friendsAdapter: ColumnAdapter<List<com.sample.Person>, ByteArray>)
+
+  data class Impl(override val _id: Int, override val name: String, override val last_name: String?,
+      override val is_cool: Boolean, override val friends: List<com.sample.Person>?,
+      override val shhh_its_secret: @Redacted String) : PersonKt
+}
+
+interface PersonKt : Person {
```

Comment:
note this is a limitation of https://youtrack.jetbrains.com/issue/KT-20068

since we don't want java code to have to call `get_id()`